bulbs/api/views.py

```diff
@@ -75,7 +81,7 @@ class ContentViewSet(UncachedResponse, viewsets.ModelViewSet):
         "authors", "types"
     )
     permission_classes = [IsAdminUser, CanEditContent]
-    metadata_class = ContentViewMetaData
+    metadata_class = InfographicMetadataMixin
```

Comment:
well I think the solution (whilst we have this dynamic view stuff), is to have some generic way of registering metadata info as necessary.